introduction.md

```diff
@@ -0,0 +1 @@
+Anthony Jackman. Up and coming nodejs/express/angularjs programmer trying to get this git thing to be second nature.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.